internal/service/cluster/resource_cluster_schema_migration_test.go

```diff
@@ -0,0 +1,131 @@
+package cluster_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/service/cluster"
+)
+
+func TestAccClusterRSClusterMigrateState_empty_advancedConfig(t *testing.T) {
+	v0State := map[string]any{
```

Comment:
I would add to tests in this file:  acc.SkipInUnitTest(t) , so they don't run in unit test, as we're not using resource.ParallelTest ... it won't have TF_ACC variable into account and will always run the tests. (it's true that these tests don't take a lot of time but still better run only when acc)

Reply:
updated